docs/serving/autoscaling/README.md

```diff
@@ -1,22 +1,17 @@
 # Autoscaling
 
-One of the main features of Knative is automatic scaling of replicas for an application to closely match incoming demand, including scaling applications to zero if no traffic is being received.
-Knative Serving enables this by default, using the Knative Pod Autoscaler (KPA).
-The Autoscaler component watches traffic flow to the application, and scales replicas up or down based on configured metrics.
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. This is provided by default, by using the Knative Pod Autoscaler (KPA).
 
-Knative services default to using autoscaling settings that are suitable for the majority of use cases. However, some workloads may require a custom, more finely-tuned configuration.
-This guide provides information about configuration options that you can modify to fit the requirements of your workload.
+For example, if an application is receiving no traffic, and scale to zero is enabled, Knative Serving scales the application down to zero pods. If scaling to zero is disabled, the application is scaled down to the minimum number of pods specified for applications on the cluster. Pods can also be scaled up to meet demand if traffic to the application increases.
```

Comment:
This is pretty unimportant and I wouldn't block this PR on it, but Im wondering a bit about the word "pods" vs "replicas".. elsewhere in the autoscaling section it looks like we've tended use the latter, I assume because "pods" is a bit of a Kubernetes implementation detail? (e.g. in https://knative.dev/docs/serving/autoscaling/scale-bounds/ and https://knative.dev/docs/serving/autoscaling/concurrency/). Maybe this is something (if it's not already in there) that we might want a style guide rule about, idk?

Reply:
should we mention `max-scale` here as well?

Reply:
+1 to using replicas instead of pods